src/content/Angular-CSharp/.template.config/ide.host.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "$schema": "http://json.schemastore.org/ide.host",
```

Comment:
@phenning was I correct to rename this file and src/content/React-CSharp/.template.config/vs-2017.3.host.json❔

Reply:
/ping @phenning

Reply:
Additional cleanup: the following can be removed from the ide.host.json file:

1. name and description properties (these will be pulled in from templatestrings.json.
2. learn more link can be removed, it is no longer used
3. uiFilters section can be removed - no longer uesed
4. supportedAuthentications can be removed - no longer used.. we just use the "auth" in the template.json
5. ports - no longer used - we use the port generators in template.json
6. excludeLaunchSetttings - no longer used
7. minFullFrameworkVersion - no longer used.

Note that the same cleanup has been done to the non-spa templates, but it looks like @DamianEdwards  and I forgot about the spa templates.